presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -47,7 +47,8 @@
         "experimental-syntax-enabled",
         "analyzer.experimental-syntax-enabled",
         "optimizer.processing-optimization",
-        "deprecated.legacy-order-by"})
+        "deprecated.legacy-order-by",
+        "deprecated.legacy-join-using"})
```

Comment:
why it is added here, if it is removed system wide?

Reply:
Hmm... probably some rebase screw up

Reply:
Ah, no, that's just the "DefunctConfig" list. It's intentional.